internal/infrastructure/http/handlers/handlers.go

```
@@ -0,0 +1,139 @@
+
+	res := ""
+	if metric.Delta != nil {
+		res = fmt.Sprintf("%v", *metric.Delta)
+	}
+
+	if metric.Value != nil {
+		res = fmt.Sprintf("%v", *metric.Value)
+	}
+
+	w.WriteHeader(200)
+	w.Write([]byte(res))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)